### PR TITLE
Homework7

### PR DESCRIPTION
**Pull request by RyslanaBinko**
Add config .travis.yml to project
Connect Travis CI to the repository
homework7: https://lms.ithillel.ua/groups/5ef4bcb08fe7c632ee2b025a/homeworks/5fcd5b25b13b6034c17f6896

### CODE REVIEW

File: dz5/home/views.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+from home.models import Student
+from home.forms import StudentForm
+
+
+def home(request):
+    if request.method == "GET":
+        student = Student()
+        student.name = "Lana"
+        student.save()
```
Comment (valeriiduz-levi9) [inline dz5/home/views.py:14]: its unneeded

Comment (valeriiduz) [inline dz5/home/views.py:14]: its unneeded

File: dz5/home/views.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        students = Student.objects.all()
+        student_form = StudentForm()
+
+        context = {"students": students,
+                   'form': student_form,
```
Comment (valeriiduz) [inline dz5/home/views.py:19]: use " " or ' ',  don't mix them

Comment (valeriiduz) [inline dz5/home/views.py:19]: and more readable style looks like:
```
context = {
        "students": students,
        "form": student_form,
}
```

**Pull request merged**